core/src/ParaGridIO.cpp

```diff
@@ -1,10 +1,11 @@
 /*!
  * @file ParaGridIO.cpp
  *
- * @date Oct 24, 2022
+ * @date 09 Dec 2024
  * @author Tim Spain <timothy.spain@nersc.no>
  */
 
+#ifndef USE_XIOS
```

Comment:
If we are only including this source file when building without XIOS, why do we need the include guards here and in `ParaGridIO_Xios.cpp`?

Reply:
Fair point, thanks @timspainNERSC. Addressed in 3463c41379d5d3c22f4f6de8f3680724153d544d and 672da0937350ad24ad48b10bce2b7e2f4b3a3b8a.